JavaScript/Endpoint Examples/JSON Payload/pdf-with-redacted-text-preview.js

```diff
@@ -0,0 +1,67 @@
+var axios = require("axios");
+var FormData = require("form-data");
+var fs = require("fs");
+
+var upload_data = fs.createReadStream("/path/to/file");
+
+var upload_config = {
+  method: "post",
+  maxBodyLength: Infinity,
+  url: "https://api.pdfrest.com/upload",
+  headers: {
+    "Api-Key": "xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx", // Replace with your API key
+    "Content-Filename": "filename.pdf",
+    "Content-Type": "application/octet-stream",
+  },
+  data: upload_data, // set the data to be sent with the request
+};
+
+// send request and handle response or error
+axios(upload_config)
+  .then(function (upload_response) {
+    console.log(JSON.stringify(upload_response.data));
+    var uploaded_id = upload_response.data.files[0].id;
+
+    var redaction_option_array = [];
+    var redaction_options1 = {
+        "type": "preset",
+        "value": "uuid",
```

Comment:
```suggestion
        "value": "email",
```

I get this error for this set of options. Replacing with `email` (or some other valid option) worked.

```
{"error": "uuid is not a valid preset redaction option. See documentation for valid options."}
```